cmd/run/run_test.go

```diff
@@ -139,7 +139,7 @@ messages:
 		_, err = runCmd.ExecuteC()
 		require.NoError(t, err)
 
-		require.Equal(t, 3, len(capturedReq.Messages))
+		require.Equal(t, 2, len(capturedReq.Messages))
```

Comment:
This is the most sus change in the whole PR, but I think it's because I fixed a bug. I checked out the version in the `main` branch, with 3 messages, and added some code to print the Messages out. This is what I got back:

```
=== RUN   TestRun/--file_pre-loads_YAML_from_file
Message 1:
  Content: You are a text summarizer.
  Role: system
Message 2:
  Content: Hello there!
  Role: user
Message 3:
  Content:
  Role: user
```

i.e. the third and final message was simply empty. So, I sheepishly just decremented this number.